src/main/java/com/yelp/nrtsearch/server/luceneserver/index/ImmutableIndexState.java

```diff
@@ -401,10 +404,16 @@ public void start(
     if (isStarted()) {
       throw new IllegalStateException("index \"" + getName() + "\" was already started");
     }
-
+    Gauge.Timer timer =
+        BootstrapMetrics.nrtsearchBootstrapTimer.labels(Version.CURRENT.toString()).startTimer();
```

Comment:
This is capturing the time to start each index, not the complete bootstrap time.